.github/workflows/checkbook_build.yml

```diff
@@ -8,6 +8,7 @@ on:
         required: true
 
 env:
+  BUILD_ENGINE: postgresql://postgres:postgres@localhost:5432/postgres
```

Comment:
You're going to run into one more issue here - these jobs/docker images don't have a running database by default. Again, go ahead and look to other workflows as examples, and ping me if you have any questions!